packages/editor/src/bindings/post-meta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../store';
+
+const { getCurrentPostType } = select( editorStore );
+
+// Prettify the name until the label is available in the REST API endpoint.
+const keyToLabel = ( key ) => {
+	return key
```

Comment:
It isn’t the most user friendly to show in the UI machine readable key that can’t be localized. Maybe it’s fine for someone technical creating the site, but far from perfect for someone editing the content.

Reply:
I agree. I saw that as a workaround. The issue is that we don't have labels in the meta fields yet. Do you think it would be better to just pass the label? `text_custom_field` for example.

Reply:
Maybe that would be more appropriate as it would be clear it’s a code name. At least until we support labels. @SaxonF, what do you think?

Reply:
I was considering using something like "Connected to { meta_key } from { source }". Which would translate into "Connected to my_custom_field from Post Meta". Would that make sense?